.changeset/flat-papayas-suffer.md

```diff
@@ -0,0 +1,171 @@
+---
+'sku': minor
+---
+
+`vite` experimental support.
+
+- Add experimental support for `vite` as a build tool.
+
+## Usage
+
+To enable `vite` as the build tool you will have to follow the steps below.
+
+### Update sku.config.ts
+
+First switch your project to `vite` by adding the following line to your `sku.config.ts`
+
+```typescript
+// sku.config.ts
+import type { SkuConfig } from 'sku';
+
+export default {
+  bundler: 'vite', // sku will use vite as the bundler.
+  ...
+} satisfies SkuConfig;
+```
+
+### Migrating your client entrypoint
+
+With `vite` the `index.html` will be generated with a different id for the root element.
+Make sure that your client entrypoint exports a function that hydrates the root element as the default export.
```

Comment:
I assume this won't be necessary once `renderDocument` is supported in vite. Maybe worth noting that this is a temporary change.